nautobot-app/{{ cookiecutter.project_slug }}/docs/dev/contributing.md

```diff
@@ -44,10 +41,31 @@ All pull requests to `next` or `develop` must include a changelog fragment file
 
 ## Branching Policy
 
-!!! warning "Developer Note - Remove Me!"
-    What branching policy is used for this project and where contributions should be made.
+The branching policy includes the following tenets:
+
+- The develop branch is the branch of the next major and minor paired version planned.
+- PRs intended to add new features should be sourced from the develop branch.
+
+{{ cookiecutter.verbose_name }} will observe semantic versioning, as of 1.0. This may result in an quick turn around in minor versions to keep pace with an ever growing feature set.
```

Comment:
```suggestion
- The `develop` branch is the branch of the next major and minor paired version planned.
- PRs intended to add new features should be sourced from the `develop` branch.
- PRs intended to fix issues in the Nautobot LTM compatible release should be sourced from the latest `ltm-<major.minor>` branch instead of `develop`.

{{ cookiecutter.verbose_name }} will observe semantic versioning, as of 1.0. This may result in a quick turnaround in minor versions to keep pace with an ever growing feature set.
```